version.properties

```diff
@@ -1,8 +1,8 @@
 #Currently building Mockito version
-version=2.15.2
```

Comment:
hmmm, this PR is picking up commits from "release/2.x" branch. Please prefer rebase vs. merge. Unless I caused it by pushing to your branch :)

Reply:
This seems to be fixed now